### DISCUSSION TITLE
Connexion LDAP

### DISCUSSION THREAD

**Discussion by rafouz21** [dbarzin/mercator#867] [Q&A]
Bonjour,

Je viens d'installer Mercator sur une VM Debian. J'aimerais en savoir plus sur la connexion LDAP. J'ai fait la connexion comme il est décrit dans la procédure d'installation, par contre je me rends compte que la connexion se fait qu'une seule. Si je comprends bien, lorsqu'un utilisateur n'est pas connu de la base Mercator il y a un check de l'AD en prenant en compte les données inscrites dans le fichier .env. Mais lorsque l'utilisateur est connu de la base, il n'y a plus de check de l'AD. Si par exemple on enlève le groupe AD à l'utilisateur, ce même utilisateur aura toujours accès à Mercator. N'y a-t-il pas un moyen pour qu'il ait toujours une requête LDAP à chaque connexion ? 
Merci de votre aide.

**Comment by dbarzin** [comment]
Cette implémentation est basée sur [LDAPRecord](https://ldaprecord.com/docs/laravel/v3)

Il existe une option qui s'appelle `LDAP_CACHE` et qui peut prendre la valeur `false` (réf.: https://ldaprecord.com/docs/laravel/v3/configuration/
